src/main/cli/cli.c

```diff
@@ -308,6 +308,7 @@ static const char *mcuTypeNames[] = {
     "H743 (Rev.V)",
     "H7A3",
     "H723/H725",
+    "G474",
```

Comment:
What about g473?

Reply:
Currently built as the same target, so we'd need custom code to determine at runtime - not sure if it is worth it, and I don't have a way of testing.

Reply:
Ok. I also don't have g473 hardware, just in case. NVM